glossary/ja/terms/linked-data/index.md

```diff
@@ -0,0 +1,7 @@
+---
+section: terms
+lang: ja
+title: Linked data
+---
+
+A form of data representation where every [identifier](/glossary/en/terms/identifier/) is an http://... {URI}, using standard lists (see [vocabulary](/glossary/en/terms/vocabulary/)) of identifiers where possible, and where datasets include links to reference datasets of the same objects. A key aim is to make [data integration](/glossary/en/terms/data-integration/) automatic, even for large datasets. Linked data is usually represented using [RDF](/glossary/en/terms/rdf/). See also [five stars of open data](/glossary/en/terms/five-stars-of-open-data/); [triple store](/glossary/en/terms/triple-store/).
```

Comment:
All the links to other terms here are linking to the English terms. Do you want to change it?